.teamcity/src/subprojects/VcsRoots.kt

```diff
@@ -45,12 +45,16 @@ object VCSAPIDocs : PasswordVcsRoot({
 object VCSKotlinxHtml : PasswordVcsRoot({
     name = "Kotlinx.html Library"
     url = "https://github.com/Kotlin/kotlinx.html.git"
+    branchSpec = """
```

Comment:
Don't we already have it somewhere? Or we simply copy-paste it over roots?

Reply:
copypaste